operator/pom.xml

```diff
@@ -0,0 +1,231 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you add information for a developer who is trying to spin this up locally?

Reply:
I will add something about this. If you get to looking at it before I do, you can try the following command from the repo root to start the operator in dev mode using your current Kube context. Assuming you have Strimzi and a `Kafka` CR in the Kube cluster, you can create a `Console` CR like in my other comment [1].

```
mvn -am -pl operator quarkus:dev
```

[1] https://github.com/streamshub/console/pull/839#issuecomment-2195612729